tests/test_utils/test_activation_checkpointing.py

```diff
@@ -62,3 +61,54 @@ def test_activation_checkpointing(cpu_offload):
     # other tests will fail if running together with this test
     # as other tests can't overwrite the seed set by this test
     reset_seeds()
+
+
+@pytest.mark.gpu
+@pytest.mark.parametrize("cpu_offload", [True, False])
+def test_activation_checkpointing_reentrant_False(cpu_offload):
+
```

Comment:
Add a `reset_seed` at the start of the function so that other tests will not affect this one.

Reply:
This applies to the test function above as well.

Reply:
oooo! I will modify it